lib/load_wallet.dart

```diff
@@ -234,6 +235,29 @@ class LoadWalletScreen extends StatelessWidget {
         child: Column(
           mainAxisAlignment: MainAxisAlignment.center,
           children: [
+            const Spacer(),
+            Consumer<WalletState>(
+              builder: (context, walletState, child) {
+                return DropdownButton<String>(
+                  hint: const Text('Select a network'),
+                  value: walletState.network,
```

Comment:
I would prefer to not use the walletState here. Just make a new State object for this screen that holds the network. Then when we create the wallet, we pass this network argument during initialization.